bin/junest

```diff
@@ -42,7 +42,8 @@ usage() {
     echo -e "-p, --proot-args <args>        Proot arguments"
     echo
     echo -e "Building options:"
-    echo -e "-b, --build-image              Build a $NAME image (must run in ArchLinux)"
+    echo -e "-b, --build-image              Build a $NAME image (must run in ArchLinux); a list of"
```

Comment:
Maybe to keep things cleaner we could use a specific argument for -b option instead of the $ARGS. For instance, similarly to $OPT_PROOT_ARGS we could have $OPT_BUILD_IMAGE_ARGS that contains the arguments for the build image option. If that would take time, no worries we can keep this solution anyway.

Also, the help could be:

    -b, --build-image  <package1,...>            Build a $NAME image (must run in ArchLinux);